src/input-select/input-select.tsx

```diff
@@ -222,6 +222,7 @@ export const InputSelect = <T, V>({
             <Selector
                 ref={selectorRef}
                 type="button"
+                aria-readonly={otherProps.readOnly}
```

Comment:
i read that aria-readonly should be applied to combo box as well. Are you saying because this selector is just a button under the hood, we dont need?

Got it on the button missing disabled. Will rectify

Reply:
Ok i took a closer read and it says to indicate an editable field is not editable. I guess in this case a button cannot be editable. I've removed the prop already

Reply:
yes, aria-readonly would be applied on the element with `role=combobox`. in most implementations, that's the div/input element that displays the selected value, while button is used here.

I'm not sure what the implications are if we set `role=combobox` on `Selector`, because that would override its implicit button role. but this is out of scope, there's additional attributes required other than the role to make it compliant anyway